CryptoPrices/Sources/Data/Sources/NetworkExtension/MyCoinsParameters.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  MyCoinsParameters.swift
+//  Data
+//
+//  Created by Doan Thieu on 09/12/2022.
+//
+
+public struct MyCoinsParameters: Parameterable {
```

Comment:
The `MyCoinsParameters` (and `Parameterable` for that matter) is actually a part of the implementation detail (`NetworkExtension`), not a part of the core/interface (`NetworkCore`). For e.g, we might have another implementation of `CoinAPIProtocol` which doesn't use any of `MyCoinsParameters` (or `Pilot`, `CoinRoute`, ...) at all. (It might just pass the parameters `perPage` or `page` directly if you got the point 🙏 ) That's why `MyCoinsParameters` belongs to `NetworkExtension`, not a separate module.